defaults/defaults_unix.go.in

```diff
@@ -21,19 +21,19 @@ package defaults
 const (
 	// DefaultRootDir is the default location used by containerd to store
 	// persistent data
-	DefaultRootDir = "/var/lib/containerd"
+	DefaultRootDir = "@CONFIG_DEFAULT_ROOT_DIR@"
```

Comment:
containerd is not just used as daemon, but also used by library.  The customized containerd binary is one common case. The change is already breaking the case...